plugins/module_utils/mso.py

```diff
@@ -1459,6 +1467,34 @@ def remove_keys_from_dict_when_value_empty(self, target_dict, modified_target=No
     def validate_schema(self, schema_id):
         return self.request("schemas/{id}/validate".format(id=schema_id), method="GET")
 
+    def input_validation(self, attr_name, attr_value, required_attributes, target_object, object_position=None, object_name=None):
+        if attr_name in (None, "") or attr_value in (None, ""):
+            self.module.fail_json(msg="The attribute and value must be set")
+
+        empty_attributes = []
+        for attribute in required_attributes:
+            if target_object.get(attribute) in (None, "", [], {}, 0):
+                empty_attributes.append(attribute)
```

Comment:
should we use list comprehension? just code syntax so will not impact the code it self

```suggestion
        empty_attributes = [attribute for attribute in required_attributes if target_object.get(attribute) in (None, "", [], {}, 0)]
```

Reply:
you added but not deleted?